MattTuring/pallet-picker-backend
# Add Project api

### What's this PR do?

This PR adds project api endpoints to developer branch and contains next changes: 

- Body Parser setup
- Get endpoints for: 
1. All projects
2. Single project
- post endpoints for:
1. Add new project
- put endpoints for:
1. Update project
- delete endpoints for:
1. Remove project

### Where should the reviewer start?

From first line of server.js file

### How should this be manually tested?

- `git fetch`
-  `git checkout project-api`
- to be sure that you have database in Postico with all information filled
- `nodemon server.js` or `npm start`
- use Postman to check api calls
